### PR TITLE
User can sign up for an account

### PR DESCRIPTION
**Pull request by rithreaksa**
- Write tests for a user can sign up an account
- Write tests for verifying the sign up form

### CODE REVIEW

File: spec/features/user_can_sign_up_for_an_account_spec.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      expect(page).to have_field 'Name'
+      expect(page).to have_field 'Email'
+      expect(page).to have_field 'Password'
+      expect(page).to have_field 'Password confirmation'
+    end
+
+    it 'display Create button' do
+      expect(page).to have_button 'Create'
+    end
+  end
```
Comment (emtalen) [inline spec/features/user_can_sign_up_for_an_account_spec.rb]: This test is not really necessary, by testing that you can fill in all of those fields you are also testing that they are there 👍🏻

File: spec/features/user_can_sign_up_for_an_account_spec.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    before do
+      visit root_path
+      click_on 'Sign up'
+    end
+    it 'is expected user to fill in the form' do
+      fill_in 'Name', with: 'Nicole'
+      fill_in 'Email', with: 'baby@gmail.com'
+      fill_in 'Password', with: '87653229'
+      fill_in 'Password confirmation', with: '87653229'
+      click_on 'Create'
```
Comment (emtalen) [inline spec/features/user_can_sign_up_for_an_account_spec.rb:19]: I'm missing an assertion, what happens after we click `Create`?
You know `expect(something).to eq something`

File: spec/features/user_can_sign_up_for_an_account_spec.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    before do
+      visit root_path
+      click_on 'Sign up'
+    end
+
+    it 'is expected user to fill in the form' do
+      fill_in 'Name', with: 'Nicole'
+      fill_in 'Email', with: 'baby@gmail.com'
+      fill_in 'Password', with: '87653229'
+      fill_in 'user_password_confirmation', with: '87653229'
```
Comment (tochman) [inline spec/features/user_can_sign_up_for_an_account_spec.rb]: change to "Password confirmation", please

File: spec/features/user_can_sign_up_for_an_account_spec.rb (hunk 1 above)
Comment (rithreaksa) [reply spec/features/user_can_sign_up_for_an_account_spec.rb]: We took away that test block.

File: spec/features/user_can_sign_up_for_an_account_spec.rb (hunk 2 above)
Comment (rithreaksa) [reply spec/features/user_can_sign_up_for_an_account_spec.rb:19]: We added the assertion.

File: spec/features/user_can_sign_up_for_an_account_spec.rb (hunk 3 above)
Comment (rithreaksa) [reply spec/features/user_can_sign_up_for_an_account_spec.rb]: Yes we did that. Thanks.

**Pull request merged**